diff --git a/drivers/media/usb/uvc/uvc_driver.c b/drivers/media/usb/uvc/uvc_driver.c
index 08fcd2ffa..75e3fc725 100644
--- a/drivers/media/usb/uvc/uvc_driver.c
+++ b/drivers/media/usb/uvc/uvc_driver.c
@@ -2445,6 +2445,16 @@ static const struct usb_device_id uvc_ids[] = {
 	  .bInterfaceSubClass	= 1,
 	  .bInterfaceProtocol	= 0,
 	  .driver_info		= (kernel_ulong_t)&uvc_ctrl_power_line_limited },
+	/* Force rev 1.0a for Quanta USB2.0 HD UVC Webcam */
+  { .match_flags          = USB_DEVICE_ID_MATCH_DEVICE
+                          | USB_DEVICE_ID_MATCH_INT_INFO,
+    .idVendor             = 0x0408,
+    .idProduct            = 0x4033,
+    .bInterfaceClass      = USB_CLASS_VIDEO,
+    .bInterfaceSubClass   = 1,
+    .bInterfaceProtocol   = UVC_PC_PROTOCOL_15,
+    .driver_info          = (kernel_ulong_t) &(const struct uvc_device_info ) {
+                            .uvc_version = 0x010a, } },
 	/* Quanta USB2.0 HD UVC Webcam */
 	{ .match_flags		= USB_DEVICE_ID_MATCH_DEVICE
 				| USB_DEVICE_ID_MATCH_INT_INFO,
