diff --git a/libfprint/drivers/elan.c b/libfprint/drivers/elan.c
index 4040060..7fb2dd8 100644
--- a/libfprint/drivers/elan.c
+++ b/libfprint/drivers/elan.c
@@ -998,8 +998,9 @@ fpi_device_elan_class_init (FpiDeviceElanClass *klass)
   dev_class->full_name = "ElanTech Fingerprint Sensor";
   dev_class->type = FP_DEVICE_TYPE_USB;
   dev_class->id_table = elan_id_table;
-  dev_class->scan_type = FP_SCAN_TYPE_SWIPE;
-
+  dev_class->scan_type = FP_SCAN_TYPE_PRESS
+;
+  img_class->algorithm = FPI_PRINT_SIGFM;
   img_class->img_open = dev_init;
   img_class->img_close = dev_deinit;
   img_class->activate = dev_activate;
diff --git a/libfprint/sigfm/meson.build b/libfprint/sigfm/meson.build
index 354d943..7b44c28 100644
--- a/libfprint/sigfm/meson.build
+++ b/libfprint/sigfm/meson.build
@@ -2,10 +2,10 @@
 sigfm_sources = ['sigfm.cpp']
 
 opencv = dependency('opencv4', required: true)
-doctest = dependency('doctest', required: true)
+# doctest = dependency('doctest', required: true)
 
 libsigfm = static_library('sigfm',
         sigfm_sources,
         dependencies: [opencv],
 )
-sigfm_tests = executable('sigfm-tests', ['./tests.cpp'], dependencies: [doctest, opencv], link_with: [libsigfm])
+# sigfm_tests = executable('sigfm-tests', ['./tests.cpp'], dependencies: [doctest, opencv], link_with: [libsigfm])
